iosApp/iosApp/Pages/StopDetails/StopDetailsFilteredDepartureDetails.swift

```diff
@@ -32,10 +29,28 @@ struct StopDetailsFilteredDepartureDetails: View {
 
     @EnvironmentObject var viewportProvider: ViewportProvider
 
+    var testTiles: [TileData]? = nil
+
     let inspection = Inspection<Self>()
 
     var analytics: Analytics = AnalyticsProvider.shared
 
+    var leafFormat: LeafFormat {
```

Comment:
question (non-blocking): Should there be an `onChange` to recalculate this if the global data changes?